playwright.config.ts

```diff
@@ -4,7 +4,7 @@ import { defineConfig, devices } from "@playwright/test";
 export default defineConfig({
     testDir: "./tests/e2e",
     /* Maximum time one test can run for. */
-    timeout: process.env.CI ? 60 * 1000 : 30 * 1000,
+    timeout: process.env.CI ? 10 * 1000 : 5 * 1000,
```

Comment:
i think these numbers were way too large, when i did a push with a couple tests that didnt work, it took ages because they also retry on CI

Reply:
I think perhaps these values went in during bad old days before you speeded up the country selection.